app/src/main/java/nl/eduvpn/app/service/PreferencesService.java

```diff
@@ -135,6 +144,19 @@ void _migrateIfNeeded(SharedPreferences newPreferences, SharedPreferences oldPre
                 Log.d(TAG, "Migrated over to storage version v3.");
             }
         }
+        if(version < 4) {
```

Comment:
You are missing a space after the `if`. You should probably reformat the code in this class.

Reply:
Fixed in https://github.com/eduvpn/android/pull/318/commits/e16a0b06986b43613fc9d8b54184350ef0d909d8